src/modules/logicModule/LM_PC_KPIRewarder_v1.sol

```diff
@@ -136,19 +135,29 @@ contract LM_PC_KPIRewarder_v1 is
     // View functions
 
     /// @inheritdoc ILM_PC_KPIRewarder_v1
-    function getKPI(uint KPInum) public view returns (KPI memory) {
+    function getKPI(uint KPInum) external view returns (KPI memory) {
```

Comment:
Why this change? Is it part of the new CoC?

Reply:
The spurious dragon came, so I had to do some size optimizations in the context of which I did some CoC changes.